vizro-core/tests/unit/vizro/actions/_callback_mapping/test_get_action_callback_mapping.py

```diff
@@ -29,51 +29,54 @@ def export_data():
 
 
 @pytest.fixture
-def managers_one_page_four_controls_three_figures_filter_interaction(request, dash_data_table_with_id):
+def config_for_testing_all_components_with_actions(request, dash_data_table_with_id):
     """Instantiates managers with one page that contains four controls, two graphs and filter interaction."""
```

Comment:
I saw that we've done the same for the unit tests for the container, so I've just replicated here. I assume the rule here is that we test for the most complex configuration within a page and this fixture is then being used in all unit tests for the actions. I was wondering if this sufficiently covers it? Or if we need to actually create different configs covering:
- Just top-level components
- Top-level containers
- Nested containers
- Tabs with containers etc.

Or is this an overkill and we always assume that if the components are visible/accessible for the actions in the most complex config (so tabs with nested containers and components inside), that it will also work for simpler configs? @petar-qb @antonymilne

Reply:
Probably we don't need to add any Containers or Tabs in this test at all (I know I've already done it for Containers), and we should test model_manager helper functions separately. 
 
However, I'm also fine by leaving this test as you have already implemented it and we can simplify this test later (by removing Container and Tabs models from it), when we implement tests for model_manager helper functions. (as a part of the https://github.com/mckinsey/vizro/issues/111)

Reply:
Agree with @petar-qb so this is fine how you have it here for now, and can simplify later as part of #111 👍